bitwarden_license/test/Commercial.Core.Test/AdminConsole/Services/ProviderServiceTests.cs

```diff
@@ -637,8 +637,7 @@ await sutProvider.GetDependency<IOrganizationService>()
                 t.First().Item1.Emails.Count() == 1 &&
                 t.First().Item1.Emails.First() == clientOwnerEmail &&
                 t.First().Item1.Type == OrganizationUserType.Owner &&
-                t.First().Item1.AccessAll &&
-                !t.First().Item1.Collections.Any() &&
+                t.First().Item1.Collections.Count() == 1 &&
```

Comment:
This test was running on pre-FC code, where the owner is granted `AccessAll`. After removing dead code, `AccessAll` no longer exists on the invite, and they always have the Default collection assigned to them instead.